### PR TITLE
Added the automation tests for PG with validations for various objects and Multiple Schemas

### PR DESCRIPTION
**Pull request by priyanshi-yb**
This patch includes the automation tests with Validation for PG for -

1. Misc-objects-2: Aggregates, Procedures, triggers, functions, extensions, and inline comments
2. Multiple Schemas

### CODE REVIEW

File: migtests/tests/pg-tests/pg-misc-objects-2/validate [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	chk_executed_query = tgt.run_query_and_chk_error("call tt_insert_data(5);", None)
+	print(f"error in executing query - {chk_executed_query}")
+	assert chk_executed_query == False
+
+	#after calling tt_insert_data() functions inserts happen on tt and audit table
+	cnt_tt = tgt.get_row_count("tt")
+	print(f"after running the procedure to insert into tt table, row count - {cnt_tt}")
+	assert cnt_tt == 5
+
+	cnt_tt = tgt.get_row_count("audit")
```
Comment (shubham-yb) [inline migtests/tests/pg-tests/pg-misc-objects-2/validate:59]: nit: can we rename the variable to match the table ex: cnt_audit

File: migtests/tests/pg-tests/pg-misc-objects-2/validate [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	res_total_proc = tgt.execute_query("select total();")
+	print(f"result of total() - {res_total_proc}")
+	assert res_total_proc == 25
+
+	INSERT_QUERY_EXT_TEST = "insert into ext_test(password) values (crypt('tomspassword', gen_salt('bf')));"
+	chk_insert_error_ext_test = tgt.run_query_and_chk_error(INSERT_QUERY_EXT_TEST, None)
+	print(f"error in inserting in ext_test - {chk_insert_error_ext_test}")
+	assert chk_insert_error_ext_test == False
+
+	cnt_tt = tgt.get_row_count("ext_test")
```
Comment (shubham-yb) [inline migtests/tests/pg-tests/pg-misc-objects-2/validate:73]: nit: can we rename the variable to match the table ex: cnt_ext_test

File: migtests/tests/pg-tests/pg-misc-objects-2/validate (hunk 1 above)
Comment (priyanshi-yb) [reply migtests/tests/pg-tests/pg-misc-objects-2/validate:59]: thanks, missed that. will do.

File: migtests/tests/pg-tests/pg-multiple-schemas/validate [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	chk_executed_query = tgt.run_query_and_chk_error("call tt_insert_data(5);", None)
+	print(f"error in executing query - {chk_executed_query}")
+	assert chk_executed_query == False
+
+	#after calling tt_insert_data() functions inserts happen on tt and audit table
+	cnt_tt = tgt.get_row_count("tt", schema_name)
+	print(f"after running the procedure to insert into tt table, row count - {cnt_tt}")
+	assert cnt_tt == 5
+
+	cnt_tt = tgt.get_row_count("audit", schema_name)
```
Comment (shubham-yb) [inline migtests/tests/pg-tests/pg-multiple-schemas/validate:60]: nit: can we rename the variable to match the table ex: cnt_audit

File: migtests/tests/pg-tests/pg-multiple-schemas/validate [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+	assert schema_set == 'schema2'
+
+	check_validations_per_schema(tgt, "schema2")
+
+	INSERT_QUERY_EXT_TEST = "insert into ext_test(password) values (crypt('tomspassword', gen_salt('bf')));"
+	chk_insert_error_ext_test = tgt.run_query_and_chk_error(INSERT_QUERY_EXT_TEST, None)
+	print(f"error in inserting in ext_test - {chk_insert_error_ext_test}")
+	assert chk_insert_error_ext_test == False
+
+	cnt_tt = tgt.get_row_count("ext_test", "schema2")
```
Comment (shubham-yb) [inline migtests/tests/pg-tests/pg-multiple-schemas/validate:96]: nit: can we rename the variable to match the table ex: cnt_ext_test

**Review by shubham-yb — APPROVED** [review]

**Pull request merged**